### PR TITLE
[WIP]: Accelerometer: update fixed point values to match datasheets for fractional digit accuracy

### PR DESCRIPTION
**Pull request by dnlup**

### CODE REVIEW

**Comment by dnlup** [comment]
@rwaldron I tried to work on `MPU6050` device but I have doubts. If and when you have some time, could you please check the comments in the changes I made to be sure I understood what you need?

File: lib/accelerometer.js [hunk 1]
```diff
@@ -79,8 +79,16 @@ var Controllers = {
     },
     toGravity: {
       value: function(raw) {
+        // Datasheet available at https://www.invensense.com/wp-content/uploads/2015/02/MPU-6000-Datasheet1.pdf
```
Comment (rwaldron) [inline lib/accelerometer.js]: When I try to visit that url:



But when I try it with "http" instead, it works just fine, so let's change the url in the comment to "http"

File: test/accelerometer.js [hunk 2]
```diff
@@ -481,6 +481,14 @@ exports["Accelerometer -- MPU-6050"] = {
...
       z: 0.8
     }]);
+    // Here I would expect this test to pass, but `digits.fractional` returns `2`.
+    // I suspect that the fact the numer is less than 1 has something to do with it.
+    // I would have tested `this.accel.x`, `this.accel.y`, `this.accel.z`
+    // but they all return a fixed value of 2 digits for the fractional part, because
+    // the Accelerometer constructor defines x,y,z properties using the `toFixed` function
+    // with a value of 2 (lines 1094-1109 in lib/accelerometer.js).
+    // Not sure what to do.
+    test.equal(3, digits.fractional(this.accel.toGravity(4369)));
```
Comment (rwaldron) [inline test/accelerometer.js]: I think we need to remove the calls to `toFixed(...)` in the getters, and let each `toGravity()` be responsible for calling `toFixed()`. Go ahead and make those changes in a different commit on this branch :)

**Comment by rwaldron** [comment]
@dnlup first: sorry for the delay in reviewing this PR. Second, your intuitions and observations were spot on; see my review notes for next steps. Ping me when ready for review!

**Comment by dnlup** [comment]
@rwaldron Thanks for reviewing this! I'll work on it and get back to you.

File: test/accelerometer.js [hunk 3]
```diff
@@ -477,18 +477,17 @@ exports["Accelerometer -- MPU-6050"] = {
...
-    // but they all return a fixed value of 2 digits for the fractional part, because
-    // the Accelerometer constructor defines x,y,z properties using the `toFixed` function
-    // with a value of 2 (lines 1094-1109 in lib/accelerometer.js).
-    // Not sure what to do.
-    test.equal(3, digits.fractional(this.accel.toGravity(4369)));
+    // Here I would expect this tests to pass, but `digits.fractional` returns the
+    // expected number of fractional digits less 1.
+    // I suspect that the fact the number is less than 1 has something to do with it,
+    // or I am missing how `digits.fractional` should work.
+    test.equal(3, digits.fractional(this.accel.x));
```
Comment (dnlup) [inline test/accelerometer.js]: `digits.fractional` returns the number of digits I would expect minus 1 (for example 2 instead of 3). It is not clear to me if  I am completely missing how the function should work.

**Comment by dnlup** [comment]
@rwaldron Made the requested changes :smile:
